--- intl/_dcigettext.c	2006-08-04 17:12:43.507997410 +0100
+++ intl/dcigettext.c	2006-08-04 17:13:26.661249829 +0100
@@ -974,18 +974,18 @@
      size_t translation_len;
 {
   struct loaded_domain *domaindata = (struct loaded_domain *) domain->data;
-  unsigned long int index;
+  unsigned long int _index;
   const char *p;
 
-  index = plural_eval (domaindata->plural, n);
-  if (index >= domaindata->nplurals)
+  _index = plural_eval (domaindata->plural, n);
+  if (_index >= domaindata->nplurals)
     /* This should never happen.  It means the plural expression and the
        given maximum value do not match.  */
-    index = 0;
+    _index = 0;
 
   /* Skip INDEX strings at TRANSLATION.  */
   p = translation;
-  while (index-- > 0)
+  while (_index-- > 0)
     {
 #ifdef _LIBC
       p = __rawmemchr (p, '\0');
